Link.java

```diff
@@ -1,22 +0,0 @@
-
```

Comment:
In general, separating the java files is only necessary if you plan to reuse the class. Seeing that this is very specific to the single linked list. It in my opinion makes more sense to contain it in the LL class.